etc/config-template.toml

```diff
@@ -161,6 +161,15 @@
 [pd]
 ## PD endpoints.
 # endpoints = []
+# The interval at which to retry a PD connection initialization. Set to 0 to disable retrying 
```

Comment:
I think 0 is meaningless. They can disable the retrying by setting `retry-max-count` to 0.